akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -314,7 +314,7 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
    * a certain size.
    */
   def registerOnMemberUp[T](code: ⇒ T): Unit =
```

Comment:
`code: => Unit` is obviously wrong if you want to except anything. There's a type for "anything" in Scala which is called `Any`. That would definitely work and you could get rid of the type parameter.